cartoframes/data/dataset.py

```diff
@@ -1,282 +1,106 @@
-                  CDB_LatLng(pickup_latitude, pickup_longitude) as the_geom,
-                  ST_Transform(CDB_LatLng(pickup_latitude, pickup_longitude), 3857) as the_geom_webmercator,
-                  cartodb_id,
-                  fare_amount
-                FROM
-                  taxi_50k
-                ''')
-
-            # show dataset on a map
-            Map(color_continuous_layer(d, 'fare_amount'))
-
-        """
-        return cls(query=query, context=context, state=cls.STATE_REMOTE, is_saved_in_carto=True)
-
-    @classmethod
-    def from_dataframe(cls, df):
-        """Create a Dataset from a local pandas DataFrame.
-
-        Args:
-          df (pandas.DataFrame): pandas DataFrame
-
-        Example:
-
-            Create a Dataset from a pandas Dataframe and then map the data.
-
-            .. code::
-
-                from cartoframes.data import Dataset
-                from cartoframes.viz import Map, Layer
-                import pandas as pd
-
-                df = pd.DataFrame({'lat': [0, 10, 20], 'lng': [20, 10, 0]})
-
-                d = Dataset.from_dataframe(df)
-
-                Map(Layer(d))
-
-        """
-        dataset = cls(df=df, state=cls.STATE_LOCAL)
-        _save_index_as_column(dataset._df)
-        return dataset
-
-    @classmethod
-    def from_geodataframe(cls, gdf):
-        """Create a Dataset from a local GeoPandas GeoDataFrame.
-
-        Args:
-          gdf (geopandas.GeoDataFrame): GeoPandas GeoDataFrame
-
-        Example:
-
-          GeoDataFrame example code taken from `GeoPandas documentation
-          <http://geopandas.org/gallery/create_geopandas_from_pandas.html#creating-a-geodataframe-from-a-dataframe-with-coordinates>`__.
-
-        .. code::
-
-            from cartoframes.data import Dataset
-            from cartoframes.viz import Map, Layer
-            import pandas as pd
-            import geopandas as gpd
-
-            df = pd.DataFrame(
-                {'City': ['Buenos Aires', 'Brasilia', 'Santiago', 'Bogota', 'Caracas'],
-                 'Country': ['Argentina', 'Brazil', 'Chile', 'Colombia', 'Venezuela'],
-                 'Latitude': [-34.58, -15.78, -33.45, 4.60, 10.48],
-                 'Longitude': [-58.66, -47.91, -70.66, -74.08, -66.86]})
-            gdf = gpd.GeoDataFrame(
-                df,
-                geometry=gpd.points_from_xy(df.Longitude, df.Latitude)
-            )
-
-            d = Dataset.from_geodataframe(gdf)
-
-            Map(Layer(d))
-
-        """
-        dataset = cls(df=gdf, state=cls.STATE_LOCAL)
-        _save_index_as_column(dataset._df)
-        return dataset
-
-    @classmethod
-    def from_geojson(cls, geojson):
-        """Create a Dataset from a GeoJSON file (hosted or local).
-
-        Args:
-          gdf (geopandas.GeoDataFrame): GeoPandas GeoDataFrame
-
-        Example:
+    GEOM_TYPE_POINT = GEOM_TYPE_POINT
+    GEOM_TYPE_LINE = GEOM_TYPE_LINE
+    GEOM_TYPE_POLYGON = GEOM_TYPE_POLYGON
```

Comment:
Yes, it is. We are using these constants in `utils` and we expose them in Dataset (for instance Dataset.GEOM_TYPE_POINT). With this solution, we are avoiding circular references. I don't see how to decrease the weird smell

Reply:
I see, let's keep it like this then. It was just assigning a variable another variable with the same name was confusing to me, but it's fine as long as it works.

Reply:
What about `import utils` (instead of cherry picking the constants and functions) and then here write:
```python
GEOM_TYPE_POINT = utils.GEOM_TYPE_POINT
# etc.
```